pkg/operator/resources/operator/operator.go

```diff
@@ -1079,3 +1092,113 @@ _The CDI Operator does not support updates yet._
 		},
 	}, nil
 }
+
+func createCDIValidatingWebhook(namespace string /*, c client.Client, l logr.Logger*/) *admissionregistrationv1beta1.ValidatingWebhookConfiguration {
+	path := "/cdi-validate"
+	sideEffect := admissionregistrationv1beta1.SideEffectClassNone
+	defaultServicePort := int32(443)
+	allScopes := admissionregistrationv1beta1.AllScopes
+	exactPolicy := admissionregistrationv1beta1.Exact
+	failurePolicy := admissionregistrationv1beta1.Ignore //FIXME: should be Fail
+	defaultTimeoutSeconds := int32(30)
+	whc := &admissionregistrationv1beta1.ValidatingWebhookConfiguration{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "admissionregistration.k8s.io/v1beta1",
+			Kind:       "ValidatingWebhookConfiguration",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cdi-api-validate",
+			Labels: map[string]string{
+				utils.CDILabel: apiServerServiceName, //FIXME?
+			},
+		},
+		Webhooks: []admissionregistrationv1beta1.ValidatingWebhook{
+			{
+				Name: "cdi-validate.cdi.kubevirt.io", //FIXME?
+				Rules: []admissionregistrationv1beta1.RuleWithOperations{{
+					Operations: []admissionregistrationv1beta1.OperationType{
+						admissionregistrationv1beta1.Create,
+						admissionregistrationv1beta1.Update,
+						admissionregistrationv1beta1.Delete,
+					},
+					Rule: admissionregistrationv1beta1.Rule{
+						APIGroups: []string{cdicorev1.SchemeGroupVersion.Group},
+						APIVersions: []string{
+							cdicorev1.SchemeGroupVersion.Version,
+							cdicorev1alpha1.SchemeGroupVersion.Version,
+						},
+						Resources: []string{"cdis"},
+						Scope:     &allScopes,
+					},
+				}},
+				ClientConfig: admissionregistrationv1beta1.WebhookClientConfig{
```

Comment:
Turns out that you can't have a webhook without tls.  I'll create a patch that bootstraps that for you

Reply:
Even with admissionregistrationv1beta1.Ignore you must have tls?
Patch will be nice:)